apps/builder/app/shared/dom-hooks/use-window-resize.ts

```diff
@@ -65,3 +65,16 @@ export const useWindowResize = (onResize: () => void) => {
     };
   }, [onResize]);
 };
+
+export const useWindowResizeDebounced = (onResize: () => void) => {
+  useEffect(() => {
+    // Call on leading
+    emitter.on("resizeStart", onResize);
```

Comment:
Can be called twice but above we have todo - add throttling, but better to make debounce with leading and trailing edges.